File: demo/hello-scripts/src/main/webapp/WEB-INF/handlebars/render.js
https://github.com/sdeleuze/spring-framework/blob/SPR-12266/spring-webmvc/src/test/resources/org/springframework/web/servlet/view/script/handlebars/render.js
function render(template, model) {
    var compiledTemplate = Handlebars.compile(template);
    return compiledTemplate(model);
}

File: demo/hello-scripts/src/main/webapp/WEB-INF/jython/render.py
# https://github.com/sdeleuze/spring-framework/blob/SPR-12266/spring-webmvc/src/test/resources/org/springframework/web/servlet/view/script/python/render.py

from string import Template

def render(template, model):
	s = Template(template)
	return s.substitute(model)